src/main/java/net/sf/mzmine/modules/peaklistmethods/io/gnpsexport/GNPSExportAndSubmitTask.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2006-2018 The MZmine 2 Development Team
+ * 
+ * This file is part of MZmine 2.
+ * 
+ * MZmine 2 is free software; you can redistribute it and/or modify it under the terms of the GNU
+ * General Public License as published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ * 
+ * MZmine 2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License along with MZmine 2; if not,
+ * write to the Free Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301
+ * USA
+ */
+/*
+ * This module was prepared by Abi Sarvepalli, Christopher Jensen, and Zheng Zhang at the Dorrestein
+ * Lab (University of California, San Diego).
+ * 
+ * It is freely available under the GNU GPL licence of MZmine2.
+ * 
+ * For any questions or concerns, please refer to:
+ * https://groups.google.com/forum/#!forum/molecular_networking_bug_reports
+ * 
+ * Credit to the Du-Lab development team for the initial commitment to the MGF export module.
+ */
+
+package net.sf.mzmine.modules.peaklistmethods.io.gnpsexport;
+
+import java.awt.Desktop;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import com.google.common.util.concurrent.AtomicDouble;
+import io.github.msdk.MSDKRuntimeException;
+import net.sf.mzmine.main.MZmineCore;
+import net.sf.mzmine.modules.peaklistmethods.io.csvexport.CSVExportTask;
+import net.sf.mzmine.modules.peaklistmethods.io.csvexport.ExportRowCommonElement;
+import net.sf.mzmine.modules.peaklistmethods.io.csvexport.ExportRowDataFileElement;
+import net.sf.mzmine.modules.peaklistmethods.io.gnpsexport.GNPSExportParameters.RowFilter;
+import net.sf.mzmine.parameters.ParameterSet;
+import net.sf.mzmine.taskcontrol.AbstractTask;
+import net.sf.mzmine.taskcontrol.AllTasksFinishedListener;
+import net.sf.mzmine.taskcontrol.Task;
+import net.sf.mzmine.taskcontrol.TaskStatus;
+import net.sf.mzmine.util.files.FileAndPathUtil;
+
+/**
+ * Exports all files needed for GNPS
+ * 
+ * @author Robin Schmid (robinschmid@uni-muenster.de)
+ *
+ */
+public class GNPSExportAndSubmitTask extends AbstractTask {
+  // Logger.
+  private final Logger LOG = Logger.getLogger(getClass().getName());
+
+  /**
+   * This website is still the beta. TODO change later to the final
```

Comment:
Is this URL ok for MZmine release?

Reply:
This website is going to be the one for a while. 
Later,  when the final website goes live, we can coordinate the transition in MZmine without loss of functionality in any time periode.